src/DynamoCore/Core/DynamoMigrator.cs

```diff
@@ -78,6 +85,33 @@ protected string CurrentVersionPath
             }
         }
 
+        protected virtual string PackagesDirectory
+        {
+            get
+            {
+                return Path.Combine(this.CurrentVersionPath, PathManager.PackagesDirectoryName);
```

Comment:
Now that the `PathManager` is updated to take `PathManagerParams`, it allows for versioned path to be generated. As disccused, the intention is to keep these path specific logic centralized inside `PathManager`, so instead of doing path combinations here locally, let's construct an instance of `PathManager` inside `DynamoMigratorBase`, and then use `IPathManager.PackagesDirectory`, `PreferenceFilePath` and `UserDefinitions` properties :)